index.bs

```diff
@@ -2170,10 +2175,31 @@ Their [=deserialization steps=], given |serialized| and |value|, are:
 
 *This section is non-normative.*
 
-This section describes the [[QUIC]] protocol behavior of methods defined
-in this specification, utilizing [[!WEB-TRANSPORT-HTTP3]]. Cause and effect may
-not be immediate due to buffering. The application
-{{WebTransportError/streamErrorCode}} in the {{WebTransportError}} error is
+This section describes the underlying protocol behavior of methods defined
+in this specification, utilizing [[!WEB-TRANSPORT-OVERVIEW]]. Cause and effect may
+not be immediate due to buffering.
+
+  <table class="data">
+    <colgroup class="header"><col></colgroup>
+    <colgroup><col></colgroup>
+    <thead>
+      <tr>
+        <th>WebTransport Protocol Action</th>
+        <th>API Effect</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>received [=session-signal/DRAIN_WEBTRANSPORT_SESSION=]</td>
```

Comment:
[DRAIN_WEBTRANSPORT_SESSION](https://pr-preview.s3.amazonaws.com/nidhijaju/webtransport/pull/562.html#session-signal-drain_webtransport_session) is it still defined in HTTP/3 it seems?
<img width="817" alt="image" src="https://github.com/w3c/webtransport/assets/3136226/2c48909a-fd1d-41cb-8c30-b15cfa1ed149">
Is that temporary?

Reply:
Yes, I filed an issue at https://github.com/ietf-wg-webtrans/draft-ietf-webtrans-http2/issues/95, so once that's resolved and it's added to the overview as well, we can reference that.